pyproject.toml

```diff
@@ -27,13 +27,14 @@ dependencies = [
     "tabulate",
     "filetype",
     "httpx",
-    "deeporigin-data-sdk==0.1.0a12",
+    "deeporigin-data-sdk==0.1.0a21",
```

Comment:
this is the thing that has triggered all of this. we were stuck on v12 for a while